server/src/test/java/io/spine/server/event/EventBusShould.java

```diff
@@ -347,6 +365,47 @@ public void create_validator_once() {
         assertSame(validator, eventBus.getValidator());
     }
 
+    @SuppressWarnings("MethodWithMultipleLoops") // OK for such test case.
+    @Ignore // This test is used only to diagnose EventBus malfunctions in concurrent environment.
+            // It's too long to execute this test per each build, so we leave it as is for now.
+            // Please see build log to find out if there were some errors during the test execution.
+    @Test
+    public void store_filters_regarding_possible_concurrent_modifications() throws InterruptedException {
+        final Thread[] threads = new Thread[50];
```

Comment:
Isn't it easier to use `Executors. ...` utility instead? 

Having an array of `Thread`s is so 2002.